notifier/notifier_test.go

```diff
@@ -80,34 +79,23 @@ func TestHandlerNextBatch(t *testing.T) {
 
 	b := h.nextBatch()
 
-	if len(b) != maxBatchSize {
-		t.Fatalf("Expected first batch of length %d, but got %d", maxBatchSize, len(b))
-	}
-	if !alertsEqual(expected[0:maxBatchSize], b) {
-		t.Fatalf("First batch did not match")
-	}
+	testutil.Equals(t, maxBatchSize, len(b))
+
+	testutil.Assert(t, alertsEqual(expected[0:maxBatchSize], b), "First batch did not match")
```

Comment:
maybe `alertsEqual` is something we want in `testutil?

Reply:
Yep , that sound reasonable.

Reply:
I think we can move the check len of 2 alert array outsite of `alertEqual` so `alertEqual` only contain the for loop.